app/helpers/application_helper.rb

```diff
@@ -1,2 +1,7 @@
 module ApplicationHelper
```

Comment:
Since `ApplicationHelper` can be a bit of a dumping ground would it make sense to extract this out to a `TextHelper` module?